.github/workflows/ci.yaml

```diff
@@ -23,11 +23,31 @@ jobs:
           cp test/buf.bash tmp/bin/buf
           chmod +x tmp/bin/buf
           echo "${PWD}/tmp/bin" >> "${GITHUB_PATH}"
-      - name: run action
+      - name: run on branch disable draft
+        if: github.ref_type == 'branch' && github.ref != 'refs/heads/main'
         uses: ./
         with:
           buf_token: fake-buf-token
           input: path/to/input
         env:
           WANT_BUF_TOKEN: fake-buf-token
-          WANT_ARGS: push --tag ${{ github.sha }} path/to/input
+          WANT_ARGS: push path/to/input --tag ${{ github.sha }}
+      - name: run on branch enable draft
+        if: github.ref_type == 'branch' && github.ref != 'refs/heads/main'
+        uses: ./
+        with:
+          buf_token: fake-buf-token
+          input: path/to/input
+          enable_draft: true
```

Comment:
to make it not breaking to existing user who push to main history on BSR from all git branch

Reply:
I can sort of see that reasoning, but also wouldn't expect that to be common.

Changing this to be on by default is a behavior change, but seems like the expected ux? With a version bump that should be fine? Or do we want to punt on that till the sync story is more clear and let this be off by default?

Reply:
yeh I was mainly not sure on how do we think about behavior change in our github actions? as user can be just tagging on version `v1` (I don't think we wanna make a `v2` because of this?), so this can be an unexpected behavior change without version bump, so I try to make this no behavior change until user specify it (although I doubt that any user is pushing from github to BSR in all branches).

Reply:
I think we should be fine to make a v2 for utilitiy libraries like this if there's significant changing functionality.

For now, lets hold off till the sync thing is more clear to turn this on automatically.